src/cgutils.cpp

```diff
@@ -2156,25 +2156,10 @@ static Value *emit_allocobj(jl_codectx_t &ctx, size_t static_size, Value *jt)
 {
     JL_FEAT_REQUIRE(ctx, dynamic_alloc);
     JL_FEAT_REQUIRE(ctx, runtime);
```

Comment:
abort compilation. currently `JL_FEAT_REQUIRE` just calls `jl_error`, and we use the `cgctx` for current function name & line number, so we'd need to do something else anyway (use DebugLoc, I assume). better leave that to a different PR, for now maybe just:

```
if (!JL_FEAT_TEST(ctx, dynamic_alloc)) jl_error(...)
```